ui/generated/gateway_pb.js

```diff
@@ -0,0 +1,1617 @@
+/**
```

Comment:
Do we need to check in generated files?

Reply:
yes for the time being. until the build process is configured.